README_SKINNY_PREFIX.rst

```diff
@@ -0,0 +1,14 @@
+=============================================
```

Comment:
Can we rename this file to `README_SKINNY`?

Reply:
of course